app/controllers/spree/admin/digital_assets_controller.rb

```diff
@@ -41,13 +42,18 @@ def spree_folder
         end
 
         def current_folder
-          @current_folder ||= Spree::Folder.find_by(id: params[:folder_id])
+          id = params.dig(:image, :folder_id) || params.dig(:folder_id)
+          @current_folder ||= Spree::Folder.find_by(id: id)
         end
 
         def current_folder_children
           @current_folder_children = current_folder.try(:children) || Spree::Folder.where(parent_id: nil)
         end
 
+        def current_digital_asset
```

Comment:
What is the advantage of this method over using the provided `@digital_asset` that you get from `ResourceController`? This seems to be what's breaking the specs.

Reply:
none. I forgot about the `@object` from `ResourcesController`. updated. thanks!